app/views/static_pages/privacy.html.erb

```diff
@@ -7,28 +7,88 @@
 <div class="row">
   <div class="col-md-12">
     <div>
-      <h2 class="fontsize-h3"><%= _("Information about you: how we use it and with whom we share it") %></h2>
-      <p><%= _("The information you provide will be used by the %{org_name} to offer you access to and personalisation of the %{application_name} service.") %
-             { application_name: ApplicationService.application_name, :org_name => Rails.configuration.x.organisation.names } %></p>
-      <p><%= _("The %{org_name} processes the personal data of %{application_name} users in order to deliver and improve the %{application_name} service in a customised manner and to ensure each user receives relevant information.") %
-             { application_name: ApplicationService.application_name, :org_name => Rails.configuration.x.organisation.name  } %></p>
-      <p><%= _("The processing of your personal data by the %{org_name} is necessary for pursuing the following legitimate interests:") %
-             { :application_name => ApplicationService.application_name, :org_name => Rails.configuration.x.organisation.name  } %></p>
```

Comment:
We should continue to use the `_()` method to handle the content translation, or are we handling this one differently?

Reply:
we don't have a french translation ready before the tri-agency meeting tomorrow. Thus Weiwei asked Alliance to use English only version for now, we will change back to the version with translation after the draft is finished

Reply:
So if I understand correctly, the text that is currently on this PR is still in the draft stage?

Reply:
Yes, we need to get the draft term of use  & privacy release before tomorrow to prepare for tri-agency meeting. That;s why you see the code here is more on 'quick fix' style. Then the privacy office (I assume) will continue to finish the term of use and privacy, and give the French translation. Then I'll change the code back to `_()` method and release again. But for now, in order to prepare for tomorrow, we will release a 'drafted' English-only version.